src/pages/FollowUpAnalysis/FollowUpAnalysisTable/FollowUpAnalysisTable.module.css

```diff
@@ -0,0 +1,5 @@
+.followUpAnalysisTable td:nth-child(4),
+.followUpAnalysisTable td:nth-child(5),
+.followUpAnalysisTable td:nth-child(6) {
+    overflow: visible !important;
+}
```

Comment:
I think I'd find this easier to work with if the overflow styles (i.e. css module class) were attached directly to the cell that they apply to. That way it's clear where styles are applied. This is more concise, but it's also easier to miss in my opinion.